### DISCUSSION TITLE
建议增加关键词屏蔽功能

### DISCUSSION THREAD

**Discussion by Jellym2** [the1812/Bilibili-Evolved#4807] [功能建议 / Ideas]
建议增加关键词屏蔽功能，在此基础上可以增加标签屏蔽等功能

**Comment by Geeyun-JY3** [comment]
https://github.com/the1812/Bilibili-Evolved/discussions/1301#discussioncomment-8787140：

> ## 不会做的功能
> 
> 以下的功能就别提功能建议了, 不会做的.
> - ……
> - 屏蔽 / 筛选满足某些条件的视频
> - ……
